databaseSetup.js

```diff
@@ -37,6 +37,9 @@ function importCreationScript(mysqlConfig, creationScriptPath) {
         if ("password" in mysqlConfig && mysqlConfig.password.length > 0) {
             params.push(`-p${mysqlConfig.password}`);
         }
+        if ("port" in mysqlConfig ) {
+            params.push(`-P ${mysqlConfig.port}`);
+        }
```

Comment:
Please add unit test coverage for this new property

Reply:
Latest change should suffice.